packages/components/src/internal/components/domainproperties/DomainRowExpandedOptions.tsx

```diff
@@ -224,6 +225,21 @@ export class DomainRowExpandedOptions extends React.Component<IDomainRowExpanded
                         lockedSqlFragment={domainFormDisplayOptions.textChoiceLockedSqlFragment}
                     />
                 );
+            case 'fileLink':
+            case 'attachment':
+                // TODO: Can this be added in LKB/LKSM?
```

Comment:
I think Hannah mentioned she was going to open a ticket for this. Hiding for the apps for now makes sense.